e-commerce-project/src/app/components/login-page/login-page.component.html

```diff
@@ -0,0 +1,8 @@
+<h3>Welcom to login page!</h3>
+<p>Enter your username and password. Then push Submit button.</p>
+
+<form (ngSubmit)="toSubmit(username.value, password.value)">
```

Comment:
1) changed name of the component to `'login-page'`
2) wrapped `<input>` tags with `<form>` tag 
3) added function `(ngSubmit)="toSubmit(username.value, password.value)"` with arguments from input fields
4) added `<button type="submit">`

**Note:** 3) and 4) allows to submit form by hitting Enter button from any place on the page (clicking the button is not required)

5) Added `"Template reference variables"` in each `<input>` field, starts hashtag (#) prefix 
 _You can refer to a template reference variable anywhere in the template_
So you can refer to the reference variable in template and even pass as an argument in function `toSubmit(username.value, password.value)`: 

_References_:
 * https://angular.io/guide/template-syntax#ref-vars
    
      or
 
 * https://angular.io/guide/user-input#get-user-input-from-a-template-reference-variable